i18n errors when running grunt locally to create a server

### Steps to reproduce:
1. Clone fxa-content-server server
2. `$ cp server/config/local.json-dist server/config/local.json`
3. `$ grunt`
### Actual results:

> Running "serverproc:test" (serverproc) task
> **fxa-content-server err:** Bad locale=[en_US] missing .key-value-json files in [/Users/pdehaan/dev/fxa-content-server_pd/fxa-content-server/app/i18n/en_US/messages.json]. See locale/README (Error: Cannot find module '/Users/pdehaan/dev/fxa-content-server_pd/fxa-content-server/app/i18n/en_US/messages.json')
> **fxa-content-server:** page_template_directory: /Users/pdehaan/dev/fxa-content-server_pd/fxa-content-server/server/templates/pages/src
> **fxa-content-server:** Firefox Account Content server listening on port 3030
### Expected results:

No error. Not sure if there is some extra config I need locally, or just a recent i18n injection.

--- a/grunttasks/default.js
+++ b/grunttasks/default.js
@@ -8,9 +8,10 @@ module.exports = function (grunt) {
   'use strict';
 
   grunt.registerTask('default', [
+    'l10n-create-json',
     'lint',
+    'copyright',
     'test',
-    'build',
-    'copyright'
+    'build'
   ]);
 };

